tests/vmi_lifecycle_test.go

```diff
@@ -1306,7 +1306,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				node := nodes.Items[0].Name
 
 				By("Creating a VirtualMachineInstance with different namespace")
-				vmi = tests.NewRandomVMIWithNS(*namespace)
+				vmi = libvmi.New(
+					libvmi.WithResourceMemory("1Mi"),
```

Comment:
Where is this coming from?
Is it due to the missing arch check? I am fine to use `256Mi` to cover it as well.

Reply:
This test does not have a running guest; any memory provided to the guest is wasted regardless of the achitecture. Should I dedicate a separate commit to explain this change, as this is *not* pure inlining?

Reply:
I see the domain is started and therefore running.
I think the memory is already assigned to the qemu process once it is started.

I would just use the minimum that fits all arch (i.e. 256Mi).

Reply:
> I would just use the minimum that fits all arch (i.e. 256Mi).

I'll do that to get this PR approved, and follow up with a separately motivated PR for memory reduction.